locksmith/src/controllers/userController.ts

```diff
@@ -1,23 +1,21 @@
 import { Request, Response } from 'express-serve-static-core' // eslint-disable-line no-unused-vars, import/no-unresolved
 import { DecoyUser } from '../utils/decoyUser'
 
-import RecoveryPhrase = require('../utils/recoveryPhrase')
 import UserOperations = require('../operations/userOperations')
 
 namespace UserController {
   export const createUser = async (
     req: Request,
     res: Response
   ): Promise<any> => {
-    let user = req.body.user
+    let user = req.body.message.user
 
     try {
       if (user) {
         let creationStatus: Boolean = await UserOperations.createUser({
           emailAddress: user.emailAddress,
           publicKey: user.publicKey,
           passwordEncryptedPrivateKey: user.passwordEncryptedPrivateKey,
-          recoveryPhrase: RecoveryPhrase.generate(),
```

Comment:
I see this is the place where you moved it one level down, right?

Reply:
yes , that is correct!